consensus/safety-rules/src/persistent_safety_storage.rs

```diff
@@ -73,10 +63,9 @@ impl PersistentSafetyStorage {
     fn initialize_keys_and_accounts(
         internal_store: &mut Storage,
         author: Author,
-        consensus_private_key: Ed25519PrivateKey,
-        execution_private_key: Ed25519PrivateKey,
+        consensus_private_key: bls12381::PrivateKey,
     ) -> Result<(), Error> {
-        let result = internal_store.import_private_key(CONSENSUS_KEY, consensus_private_key);
+        let result = internal_store.set(CONSENSUS_KEY, consensus_private_key);
```

Comment:
import_private_key API is only for Ed25519 and it's hard to change because the underlying vault implementation only supports Ed25519 as crypto storage